CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ yarn lint
 - Update the package version in the `package.json` file
 - Run `yarn clean && yarn build` to generate a new build
 - Check manually the content of the `build` directory
-- Run `yarn publish` to publish the package
+- Run `yarn npm publish` to publish the package
```

Comment:
Isn't yarn supposed to be an alternative to npm? Why do we need both here?

Reply:
Yarn is an alternative to npm CLI. 
npm is much more than its CLI, it is also the registry for node packages: https://www.npmjs.com/about:) 
This commands means that we publish the package to npm registry: https://yarnpkg.com/cli/npm/publish

I believe all yarn commands relative to the npm registry where moved under the `npm` command to make it clearer.